app/javascript/react/containers/ReviewIndexContainer.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { browserHistory, Link } from 'react-router';
+import ReviewShowTile from '../components/ReviewShowTile'
+
+class ReviewIndexContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: [
+        {
+          id: 1,
+          user: "sdljksdljfklsf",
+          description: "jsdkldjkasda",
+          price: 2,
+          service: 1,
+          reliability: 2,
+          overall: 3
+
+        }, {
+          id: 2,
+          user: "fkdslfkds;f",
+          description: "fofksdl;fkls",
+          price: 2,
+          service: 2,
+          reliability: 3,
+          overall: 1
+
+        }, {
+          id: 3,
+          user: "Mike",
+          description: "description",
+          price: "2",
+          service: "1",
+          reliability: "2",
+          overall: "1"
+        }
+      ]
+    }
+  }
```

Comment:
this component is not ready
- the default state shouldnt be hardcoded, it should be blank
- at no point are the reviews being pulled from the backend. 
- In order to get the reviews, you actually should serialize the reviews in with the fetch request you already have in your show container